Fachportal Padagogik.js

```diff
@@ -0,0 +1,331 @@
+				"issue": "1",
+				"itemID": "article",
+				"libraryCatalog": "Fachportal Pädagogik",
+				"pages": "176-197",
+				"publicationTitle": "Journal for educational research online",
+				"volume": "7",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					},
+					{
+						"title": "Link",
+						"snapshot": false
+					}
+				],
+				"tags": [
+					"Deutschland",
+					"Empirische Untersuchung",
+					"Fragebogen",
+					"Lehrer",
+					"Lernförderung",
+					"Lernmethode",
+					"Mathematikunterricht",
+					"Schuljahr 09",
+					"Selbstgesteuertes Lernen",
+					"Unterrichtsforschung",
+					"Überzeugung"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.fachportal-paedagogik.de/fis_bildung/suche/fis_set_e.html?FId=A18490&mstn=3&ckd=no&mtz=100&facets=y&maxg=5&suche=erweitert&ohneSynonyme=y&feldname1=Freitext&feldinhalt1=MATHEMATIK&bool1=and&nHits=19233&next=A18569,A19036,1065678&prev=A18195,A18196&marker=1",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "10 Jahre international vergleichende Schulleistungsforschung in der Grundschule. Vertiefende Analysen zu IGLU und TIMSS 2001 bis 2011.",
+				"creators": [
+					{
+						"firstName": "Heike ",
```

Comment:
if it's even a problem. As I say, I believe the trailing spaces are actually ignored on import, just better to check.
beyond that
`{Wendt, Heike  and Stubbe, Tobias C.  and Schwippert, Knut  and Bos, Wilfried }`
seems pretty rare to me, though yes, it'd be an easy fix in bibtex.